CHANGELOG.md

```diff
@@ -281,3 +281,6 @@ Additions to existing modules
   ↭-sym       : Symmetric (Vector A) _↭_
   ↭-trans     : Transitive (Vector A) _↭_
   ```
+
+* `Tactic.Cong` now provides a marker function, `⌞_⌟`, for user-guided
+  anti-unification.
```

Comment:
Maybe add `See README.... for details`?

Reply:
Yes. Let's make this more discoverable. Done.